### PR TITLE
Objedetect: Fix dataset: russian.jpg

### PR DESCRIPTION
**Pull request by allnes**

### CODE REVIEW

File: testdata/perf/objdetect.xml [hunk 1]
```diff
@@ -705,13 +704,13 @@
...
+      <x>1</x>
+      <y>13</y>
       <val>255.</val></rng1>
     <rng2>
-      <x>7</x>
-      <y>3</y>
-      <val>255.</val></rng2></straight_barcode></Perf_Objdetect_QRCode--decode---version_1_down-jpg->
+      <x>10</x>
+      <y>4</y>
+      <val>0.</val></rng2></straight_barcode></Perf_Objdetect_QRCode--decode---version_1_down-jpg->
```
Comment (alalek) [inline testdata/perf/objdetect.xml]: Too many unrelated changes.

Comment (allnes) [reply testdata/perf/objdetect.xml]: @alalek fixed
